docker/venice-client/sample-data/multi-dc-configs/batch-push-job.properties

```diff
@@ -0,0 +1,15 @@
+venice.urls=http://venice-controller.dc-parent.venicedb.io:5555
+venice.discover.urls=http://venice-controller.dc-parent.venicedb.io:5555
+venice.store.name=test-store
+input.path=/opt/venice/sample-data/batch-push-data
+key.field=key
+value.field=value
+controller.request.retry.attempts=5
+venice.writer.close.timeout.ms=500
+ssl.trust.store.property.name=test
```

Comment:
The ssl configs are now optional. All 4 can be removed, if SSL is not enabled.

Reply:
Ack